mobile/README.md

```diff
@@ -19,3 +19,9 @@ dart pub global activate flutterfire_cli
 ```bash
 flutterfire configure
  ```
+
+ ## In order to generate the apk file, you need go to mobile/scc_poc_app folder then run the command:
```

Comment:
## Generate the APK file

Start from root folder, navigate to `mobile/scc_poc_app` folder then run the flutter build command:

 ```bash
cd mobile/scc_poc_app
flutter build apk
 ```
The release apk file will be generated at `mobile/scc_poc_app/build/app/outputs/flutter-apk/app-release.apk`